packages/hot-fuzz/README.md

```diff
@@ -0,0 +1,117 @@
+Hot Fuzz
+========
+
+> Testing Superfluid protocol and Super Apps like Simon Pegg.
+
+<center>
+    <img src="hot-fuzz-simon.jpg" width="61.803%" />
+</center>
+
+Hot-fuzz is a wrapper of [Echidna documentation](https://github.com/crytic/echidna/) with additional helper for fuzzing
+your Superfluid smart contracts applications, including [Super Apps](https://docs.superfluid.finance/superfluid/protocol-developers/super-apps).
+
+How To Use
+==========
+
+## Setup
+
+1. Download the latest echidna binary from: https://github.com/crytic/echidna
+
+> :warning: Temporary Workaround needed
+> The hot-fuzz currently needs a fix in echidna to support contract deployment to specific address.
+>
+> The fix can be found at https://github.com/crytic/echidna/pull/758.
+>
+> To download, open the link above -> click "Checks" tab -> click "CI" -> Download an artifact suitable for your
+> operating system.
+
+2. Add `@superfluid-finance/hot-fuzz` to your project `devDependencies`.
+
+`hot-fuzz` does not have a package yet, you should install it through:
+
+```
+yarn add https://gitpkg.now.sh/superfluid-finance/protocol-monorepo/packages/hot-fuzz?echidna_fuzzing_integratio
```

Comment:
this link is broken, it's missing `n`.
fix: `https://gitpkg.now.sh/superfluid-finance/protocol-monorepo/packages/hot-fuzz\?echidna_fuzzing_integration`

Reply:
I guess it should actually be `...?dev`, since after merging this branch becomes obsolete

Reply:
also:
```
didi@P1G4:~/src/sf/protocol-monorepo-hotfuzz/examples/flowlottery/solidity-contracts$ yarn install
yarn install v1.22.11
[1/4] Resolving packages...
[2/4] Fetching packages...
error https://gitpkg.now.sh/superfluid-finance/protocol-monorepo/packages/hot-fuzz?echidna_fuzzing_integration: Integrity check failed for "@superfluid-finance/hot-fuzz" (computed integrity doesn't match our records, got "sha512-b2irPMsf/ErEgNJ9TXLkEbAV6mJhfDAc3v7Be+8Z1QPOWiTXJoVVEXhIBV8awUepafnbFFCXGfeu5zWc6iAmzg== sha1-BRyE9K4DPVNAuF6sMwKkcvGLv1w=")
```

Reply:
the lockfile was not correctly generated I guess, it should work otherwise, I can check it.

But there is a chicken and egg problem at the moment, because it's not in the dev yet. We can fix it in a small follow up pr once it's it merged

Reply:
found a immutable link for package.json:

        "@superfluid-finance/hot-fuzz": "https://gitpkg.now.sh/superfluid-finance/protocol-monorepo/packages/hot-fuzz?echidna_fuzzing_integration#c5e36f824c5b03e4be297874da2f92a5a6f4fd87",

will still need to do a PR after merging to dev

Reply:
try again @d10r @0xdavinchee to install from flowlottery solidity-contracts folder

Reply:
still getting that error :-/

but if another PR is needed anyway, we may ignore this for now

Reply:
try again with the flowlottery solidity contracts I changed how the dependency is :

        "@superfluid-finance/hot-fuzz": "https://gitpkg.now.sh/api/pkg?url=superfluid-finance/protocol-monorepo/packages/hot-fuzz&commit=da8234b58cf614922936e15c1befe3944e008c12",

Reply:
once merged, I will change that to a dev branch git commit hash